src/features/geometry_uploader/index.tsx

```diff
@@ -52,15 +51,9 @@ export function initFileUploader() {
           { type: 'uploaded' },
           geoJSON,
         );
-        const map = currentMapAtom.getState();
-        if (!map) return;
 
-        bbox.length = 4;
-        const camera = map.cameraForBounds(bbox, {
-          padding: app_config.autoFocus.desktopPaddings,
-        });
-        if (!camera) return;
-        const { zoom, center } = camera;
+        if (!geometryCamera) return;
```

Comment:
i would expect to see this early return right before` if (typeof geometryCamera === 'string') {`

Reply:
fixed